roofit/roofitcore/src/RooFactoryWSTool.cxx

```diff
@@ -2077,11 +2077,17 @@ std::string RooFactoryWSTool::SpecialsIFace::create(RooFactoryWSTool& ft, const
         vector<string> subargs = ft.splitFunctionArgs(pargv[i].c_str()) ;
         if (subargs.size()==1){
           string expr = ft.processExpression(subargs[0].c_str());
-          strlcat(pargsmorph, subargs[0].c_str(),BUFFER_SIZE);
+          for(auto param:func_args){
```

Comment:
```suggestion
          for(auto const& param:func_args){
```
It's a good habit to use `const&` by default, to avoid copying (here you copy strings unnecessarily, for example).